src/lib.rs

```diff
@@ -93,11 +95,33 @@ impl fmt::Display for Repository {
             write!(f, " - {}", description)?;
         }
 
+        if let Some(ref languages_url) = self.languages_url {
+            write!(f, " - {}", languages_url)?;
+        }
+
         Ok(())
     }
 }
 
-pub fn collect_stars(config: Config) -> Result<(), Box<dyn error::Error>> {
+#[derive(Debug, Deserialize)]
+struct Languages {
+    key: Option<String>,
+    value: Option<String>,
+}
+
+impl fmt::Display for Languages {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        if let Some(ref key) = self.key {
+            write!(f, " - {}", key)?;
+        }
+         if let Some(ref value) = self.value {
+            write!(f, " - {}", value)?;
+        }
+        Ok(())
+    }
+}
+
+pub fn collect_stars(config: Config) -> Result<(), Box<dyn error::Error>> { //Where the json extraction happens
```

Comment:
```suggestion
pub fn collect_stars(config: Config) -> Result<(), Box<dyn error::Error>> {
```

nit: extraneous comment